src/tourmaline/client/core_methods.cr

```diff
@@ -198,28 +198,31 @@ module Tourmaline
         })
       end
 
-      # Use this method to edit text and game messages. On success, if
-      # edited message is sent by the bot, the edited `Message`
-      # is returned, otherwise `true` is returned.
+      # Use this method to edit text and game messages.
+      # On success, if the edited message is not an inline message,
+      # the edited Message is returned, otherwise True is returned.
       def edit_message_text(
-        chat,
         text,
+        chat = nil,
         message = nil,
         inline_message = nil,
         parse_mode = @default_parse_mode,
         entities = [] of MessageEntity,
         disable_link_preview = false,
         reply_markup = nil
       )
-        if !message && !inline_message
-          raise "A message_id or inline_message_id is required"
+        if (!message || !chat) && !inline_message
+          raise "A message_id witth chat_id or inline_message_id is required"
```

Comment:
Please change the error message to `"edit_message_text requires either a chat and a message, or an inline_message"`